RecoEgamma/EgammaMCTools/python/pfClusterMatchedToPhotonsSelector_cfi.py

```diff
@@ -0,0 +1,10 @@
+import FWCore.ParameterSet.Config as cms
+
+particleFlowClusterECALMatchedToPhotons = cms.EDProducer('PFClusterMatchedToPhotonsSelector',
```

Comment:
No: please stick on fillDescription and remove this file.
Explicit python config can be used if you want to clone and modify the default module